benchmarks/bench_sm103_vs_sm100.py

```
@@ -0,0 +1,431 @@
+    alpha = torch.tensor(
+        [1.0 / (a_gsf.item() * b_gsf.item())],
+        dtype=torch.float32,
+        device=device,
+    )
+    # mm_fp4 API convention: b is (k_packed, n), b_descale is (k_sf, n_sf)
+    return a_fp4, b_fp4.T, a_sf, b_sf.T, alpha
+
+
+# -- Tactic helpers -----------------------------------------------------------
+def format_tactic(tactic):
+    mma, cluster, swap, prefetch, ktype, tma_store = tactic
```

Ruff (0.16.3)

[warning] Unpacked variable `prefetch` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)